autorally_description/urdf/autoRallyPlatform.urdf.xacro

```diff
@@ -415,14 +435,14 @@ limitations under the License.
   </link>
 
   <joint name="imu_joint" type="fixed">
-    <origin xyz="0 0 0" rpy="0 0 0"/>
+    <origin xyz="-0.039 0 0.0859" rpy="0 0 0"/>
```

Comment:
Please verify that MPPI can still drive the car with the state estimator in simulation with this change. We've had previous issues with the imu plugin not properly taking into account transformations, which is why in the state estimator launch file you have to invert two axis for simulation.

Reply:
MPPI can still drive the car in simulation